### ISSUE TITLE
Output normalization in dualrnn_test_wav.py

### ISSUE THREAD

**Issue by R7788380** [JusperLee/Dual-Path-RNN-Pytorch#47]
Thank you for your contribution !

I have a question in the [line 44](https://github.com/JusperLee/Dual-Path-RNN-Pytorch/blob/master/dualrnn_test_wav.py#L44) of dualrnn_test_wav.py.

You normalize the DPRNN prediction like below:
```
norm = torch.norm(egs,float('inf'))
#norm
s = s - torch.mean(s)
s = s*norm/torch.max(torch.abs(s))
```

I'm a little confused about here, because you don't do any input normalization in training pipeline, why you do here?

**Comment by JusperLee** [comment]
I do this because if I use torchaudio to generate audio truncation may occur.
You also can use scipy to generate estimated audio.

**Comment by R7788380** [comment]
I don't understand why truncation occurs when using torchaudio to generate(`torchaudio.save`) audio.
Can you give an example?

So if I use scipy or some other function (ex. soundfile) I can avoid truncation?

**Comment by JusperLee** [comment]
I think you can use the following code without any questions.
```python
from scipy.io import wavfile
wavfile.write('a.wav', 8000, np.asarray(data, dtype=np.int16)
```

**Comment by R7788380** [comment]
Thank you so much!
